src/types/Newsroom.ts

```diff
@@ -69,6 +69,7 @@ export interface NewsroomRef {
         manual_subscription_management: string;
         privacy_settings: string;
         widget_settings: string;
+        widget_javascript: string;
```

Comment:
@aspirisen this is the widget javascript URL, e.g.: https://bea-theme.prezly.com/widget/javascript

Is that the one you need here?

Reply:
i am not sure, i do not need the javascript itself. I need the feed layout, like that `https://bea-theme.prezly.com/feed.widget`

Reply:
Got it, changed! 👍🏻

Reply:
@marko-ogg wait a bit, let's discuss in the linear ticket, maybe that would be enough?